Encounter Me/Pages/TrailProgress.razor

```diff
@@ -43,6 +43,8 @@
     private Modal _infoModal = new Modal();
 
     private UserData currentUser;
+    private UserData userDataSnapshot;
+    private CapturePoint _capturePointInProgressSnapshot;
```

Comment:
It is hard to rename fields in .razor files later.  Why `currentUser` and `userDataSnapshot` do not follow naming convntions? If I remember correctly we agreed to have _camelCase for private fields. If not now, I think you should  change it in `update/Cleanup` branch.